ynr/apps/popolo/models.py

```diff
@@ -389,6 +389,16 @@ def name_and_party(self):
         """
         return f"{self.person.name} ({self.party.name})"
 
+    @property
+    def previous_party_affiliations_string(self):
+        """
+        Return a comma seperated list of EC-ID's for previous party affiliations
+        relations
+        """
+        return ", ".join(
```

Comment:
When I had tested locally this wasn't an issue:
![Screenshot 2022-03-18 at 12 04 16](https://user-images.githubusercontent.com/15347726/158999902-c3abb530-7a52-4dba-9c7b-119775c40d7a.png)

I think because it is returned as a string when there are multiple values like: `"PP851, PPm12607, PP12653"`

Still think its best to change the delimiter to be on the safe side though so have made that change